src/middleware.js

```diff
@@ -9,17 +9,40 @@ function durationMiddleware() {
     const labels = {
       method: ctx.method,
       code: ctx.status,
-      // eslint-disable-next-line no-underscore-dangle
-      route: ctx._matchedRoute,
+      route: ctx.request.url,
     };
     stats.timing('http_request_duration', startTime, labels);
   };
 }
 
+function requestSizeMiddleware() {
+  return async (ctx, next) => {
+    await next();
+
+    const labels = {
+      method: ctx.method,
+      code: ctx.status,
+      route: ctx.request.url,
```

Comment:
Explain: Changed from ctx._matchedRoute to ctx.request.url. This is done to capture route of all the requests received.